test/Generators/Microsoft.Gen.Logging/Generated/Microsoft.Gen.Logging.Generated.Tests.csproj

```diff
@@ -11,6 +11,7 @@
     <InjectIsExternalInitOnLegacy>true</InjectIsExternalInitOnLegacy>
     <InjectSharedNumericExtensions>true</InjectSharedNumericExtensions>
     <NoWarn>$(NoWarn);IDE0161;S1144</NoWarn>
+    <UseLoggingGenerator>true</UseLoggingGenerator>
```

Comment:
This should not be needed here.

Reply:
You're right. I was trying to get the changes from the updated parsing logic to take hold. All I needed to do was clean + reload solution.